### DISCUSSION TITLE
How to locally generate markdown for dashboard or markdown for a specific pull request

### DISCUSSION THREAD

**Discussion by nitrocode** [renovatebot/renovate#31377] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

There no option above for running renovate locally. I'm enabling debug and setting platform to local. I don't see the renovate dashboard markdown nor the pr markdown in the output when running locally.

```bash
docker run --rm -ti \
  -e LOG_LEVEL=debug \
  -e GITHUB_COM_TOKEN="$GITHUB_COM_TOKEN" \
  -v /tmp:/tmp \
 -v $(pwd):/usr/src/app \
```

Is there an option for that? If not, could this be a feature request? 

At the moment, I'm using the GitHub managed app and I'm shotgunning changes (pr to central config, get approval, merge, see if the changes worked as expected).

Thanks!

### Logs (if relevant)

n/a

**Comment by RahulGautamSingh** [comment]
No PRs are created when running in local mode. You can only check extracted Deps and lookups.

**Reply by nitrocode** [reply]
Thanks Rahul. Can there be a feature request to show this mark down for issue dashboard and prs when running in local mode?

**Reply by RahulGautamSingh** [reply]
You should setup a test repo or fork your main repo and run on that.

**Reply by RahulGautamSingh** [reply]
> Can there be a feature request to show this mark down for issue dashboard and prs when running in local mode?

I don't think this feature will be very desirable. We do not log that info ie. the `prBody`. As the bodies are big and will only cause trouble when debugging. If you want you can run the renovate on `dryRun=full` and put some console logs in the code
